### PR TITLE
buckshot type and canister added. now to figure

### PR DESCRIPTION
**Pull request by KoishiVibe**
# Summary
- M108 Canister ammunition.
- creates space
- gratefully carried by vero5123

### CODE REVIEW

**Comment by KoishiVibe** [comment]
aw what the fuck

File: code/game/objects/items/storage/boxes.dm [hunk 1]
```diff
@@ -631,6 +631,17 @@
 	grenade_type = /obj/item/explosive/grenade/phosphorus
 	has_gamemode_skin = FALSE
 
+/obj/item/storage/box/nade_box/canister
+	name = "\improper M108 Canister Grenade Box"
```
Comment (XSlayer300) [inline code/game/objects/items/storage/boxes.dm:635]:
```suggestion
	name = "\improper M108 canister grenade box"
```

**Comment by KoishiVibe** [comment]
And with that, it should, hopefully, work.

**Review by morrowwolf — CHANGES_REQUESTED** [review]
I would like to think about this before merged. Undraft when you're ready.

**Comment by AndroBetel** [comment]
https://github.com/PvE-CMSS13/PvE-CMSS13/pull/192

**Pull request closed without merging**